<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>망등록</string>
  </property>
  <property name="toolTip">
   <string>록</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>385</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PB_CheckMangNo">
        <property name="toolTip">
         <string>망번호 중복체크가 되어야지 저장할 수 있습니다.</string>
        </property>
        <property name="text">
         <string>망번호 중복체크</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>망번호 중복체크가 되어야지 저장할 수 있습니다.</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_Form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>망번호</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_MangNo">
          <property name="placeholderText">
           <string>채번규칙:22-001(년도2자리-Serial 자리)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>고객사</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="고객사_Widget" name="widget_Gogaek" native="true"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>현장명</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_Hyunjang"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>문양</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_Munyang"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>의장종류</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="Widget_망관리_의장종료" name="widget_Uljang" native="true"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>할부치수</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="Widget_망관리_할부치수" name="widget_Halbu" native="true"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>품명</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="Widget_망관리_품명" name="widget_Pummyung" native="true"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>망사</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="Widget_망관리_망사" name="widget_Mangsa" native="true"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>사용구분</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="Widget_망관리_사용구분" name="widget_SayongGubun" native="true"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>세부내용</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QPlainTextEdit" name="plainTextEdit_Sebu"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>비고</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QPlainTextEdit" name="plainTextEdit_Bigo"/>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>검색Key</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLineEdit" name="lineEdit_Gumsaek">
          <property name="placeholderText">
           <string>형식은 검색어,검색어 형태입니다.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_Img">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ImageViewer2" name="widget_Image1" native="true"/>
      </item>
      <item>
       <widget class="ImageViewer2" name="widget_Image2" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>고객사_Widget</class>
   <extends>QWidget</extends>
   <header>modules.PyQt.component.combo_lineedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget_망관리_의장종료</class>
   <extends>QWidget</extends>
   <header>modules.PyQt.component.combo_lineedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget_망관리_할부치수</class>
   <extends>QWidget</extends>
   <header>modules.PyQt.component.combo_lineedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget_망관리_품명</class>
   <extends>QWidget</extends>
   <header>modules.PyQt.component.combo_lineedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget_망관리_망사</class>
   <extends>QWidget</extends>
   <header>modules.PyQt.component.combo_lineedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget_망관리_사용구분</class>
   <extends>QWidget</extends>
   <header>modules.PyQt.component.combo_lineedit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageViewer2</class>
   <extends>QWidget</extends>
   <header location="global">modules.PyQt.component.imageViewer2</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
